packages/client/e2e/pages/CreditRfqTilesPageObject.ts

```diff
@@ -0,0 +1,43 @@
+import { Page } from "@playwright/test"
+
+export class CreditRfqTilesPageObject {
+  constructor(readonly page: Page) {}
+
+  // TODO - these tiles filter functions will not work in squashed resonsive mode
```

Comment:
.. broke my own rule 🤦‍♂️ .. so here's a comment to cover it as we are doing the work straight away:
RFQ Tiles filter in responsive mode - SU-68